src/sdl/sdl.h

```diff
@@ -21,6 +21,6 @@ along with this program; if not, write to the Free Software
 Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
 */
 
-#include <SDL.h>
+#include <SDL2/SDL.h>
```

Comment:
We include the SDL2 include directory, so this change is not needed.

Reply:
I was looking at the docs. It  it's unclear which one to pick. Both are available on my system. I'll revert. cmake should make it prefer SDL2, instead of any potential /usr/include/SDL.h